pages/about/editorial_board.md

```diff
@@ -25,7 +25,7 @@ Being on the editorial board of ELIXIR Training SPLASH entails shaping how Train
 * Provide timely feedback for improving the quality of the content, according to the style guide.
 * Formatting content according to templates.
 * Join the <regular> editorial meeting.
```

Comment:
I'm not sure if the highlighting of the word `regular` would work, so to be safe, suggest removing it.

```suggestion
* Join the regular editorial meeting.
```